ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js

```diff
@@ -20,6 +20,7 @@ export default class AdvancedGasInputs extends Component {
     isSpeedUp: PropTypes.bool,
     customGasLimitMessage: PropTypes.string,
     minimumGasLimit: PropTypes.number,
+    customPriceIsExcessive: PropTypes.bool,
```

Comment:
Nit: Could you add a default of `false` here, so that it's more clear that this is deliberately left optional?

Since this component is used in a variety of different places, leaving this as optional does seem appropriate.

Reply:
done